src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -60,6 +63,49 @@ class MediaBaseViewer extends BaseViewer {
         this.loadTimeout = 100000;
         this.oldVolume = DEFAULT_VOLUME;
         this.pauseListener = null;
+
+        this.startTimeInSeconds = this.getStartTimeInSeconds();
```

Comment:
Ah I see, you need to set it as a property in MediaBase. In that case, it may be worth keeping consistency in the doc viewer.

Then the doc logic becomes:
`const startPage = this.startPageNum || this.getCachedPage();`